indexing-engine/cp-model/goals.mdx

```diff
@@ -14,35 +16,61 @@ Further, some goals should not be selected as the first goals to optimize.
 
 For example, suppose that the first goal is [Minimize Number of Indexes](#minimize-number-of-indexes). The model is essentially told that "use the fewest indexes" is the most important goal. Obviously, 0 indexes is the fewest indexes it can use. Subsequent goals will not be able to make use of any indexes because of this.
 
-Goals that should be avoided as the first goal are [Minimize Number of Indexes](#minimize-number-of-indexes), [Minimize Index Write Overhead](#minimize-index-write-overhead), and [Minimize Update Overhead](#minimize-update-overhead), unless there is a [rule](rules) that is focused on the scan cost, like [Minimum Per-Scan Cost (Normal)](rules#minimum-per-scan-cost-normal).
+Goals that should be avoided as the first goal are [Minimize Number of Indexes](#minimize-number-of-indexes), [Minimize Index Write Overhead](#minimize-index-write-overhead), and [Minimize Update Overhead](#minimize-update-overhead), unless there is a [rule](rules) that is focused on the scan cost, like [Maximum Per-Scan Cost Tolerance](rules#maximum-per-scan-cost-tolerance).
 
 
 ### Minimize Index Write Overhead
 
 The *Minimize Index Write Overhead* goal strives to minimize the index write overhead associated with the selected indexes.
 
+<CopyCodeBlock content={`{ "Name": "Minimize Index Write Overhead", "Tolerance": 0.0 }`} language="json" />
```

Comment:
FWIW, it appears the syntax highlighting isn't working here yet - not sure what exactly is going on.

Reply:
Yeah, I've noticed that elsewhere. I don't think it's a blocker for this, but I'll see if I can track that down.